### ISSUE TITLE
Multiple colours for triangles

### ISSUE THREAD

**Issue by sakurazomu** [eclipsedteam/osu-trigen#18]
Such as http://puu.sh/GABvu/cc48a3ed02.jpg
If there is no hope to join. Like this app very much.

**Comment by Tanza3D** [comment]
As of right now that isn't a feature, sadly. I'll see what I can do, though
no promises

**Comment by sakurazomu** [comment]
Thank you for your efforts! Also hope to support full screen display.

**Comment by Tanza3D** [comment]
> 
> 
> Thank you for your efforts! Also hope to support full screen display.

if you want to fullscreen the triangles, you can use ALT + ENTER!

**Comment by Tanza3D** [comment]
how's this look @sakurazomu 
![image](https://user-images.githubusercontent.com/33783503/95638656-4dfc9f00-0a8d-11eb-8c5d-48a472429d1e.png)

**Comment by Tanza3D** [comment]
the colours in the screenshot i sent are preset & forced, if you need full control over these i can provide that, though not easily

**Comment by sakurazomu** [comment]
Yes, it looks great, very gorgeous. But if this is not easy, then don't bother you.
